scripts/export.py

```diff
@@ -57,7 +59,25 @@ def export_era5POS():
     exporter.export(variable='precipitation_amount_1hour_Accumulation')
 
 
+def export_gleam():
+    # if the working directory is alread ml_drought don't need ../data
+    if Path('.').absolute().as_posix().split('/')[-1] == 'ml_drought':
+        data_path = Path('data')
+    else:
+        data_path = Path('../data')
+
+    host = 'Get the login details'
```

Comment:
@tommylees112 , it seems like the login details are uniform (the ones you got were the same as what I got); we could just add them to the GLEAM exporter class.

Reply:
yeah i think we should - can you make that change? or add to the script example?

Reply:
I'll make the change!

Reply:
Updated